content/en/integrations/guide/storage-monitoring-setup.md

```diff
@@ -18,9 +18,37 @@ Access your Storage Monitoring data by navigating to **Infrastructure > Storage
 ### Installation
 
 {{< tabs >}}
+{{% tab "Recommended: Storage Monitoring UI" %}}
+
+The fastest way to set up Storage Monitoring is going to Infrastructure -> Storage Monitoring -> [Add Buckets][https://app.datadoghq.com/storage-monitoring?mConfigure=true]. In the Add Buckets page you can configure multiple S3 buckets for Storage Monitoring in one go.
```

Comment:
```suggestion
The fastest way to set up Storage Monitoring is going to **Infrastructure > Storage Monitoring > [Add Buckets][https://app.datadoghq.com/storage-monitoring?mConfigure=true]**. On the Add Buckets page, you can configure multiple S3 buckets for Storage Monitoring in one go.
```
Please use [numbered reference-style links](https://www.markdownguide.org/basic-syntax/#reference-style-links) -- this is just to support proper translation.